src/Components/DogCardTemplate/DogCardTemplate.styles.scss

```diff
@@ -0,0 +1,15 @@
+.card {
+    width: 300px;
+    margin: 10px;
+    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+}
+.card-img-top {
+    width: 100% !important;
```

Comment:
why !important ?
In general, we try to avoid using important if it is not necessary. 
Usually, you might need them to override 3rd part libraries but in your case i don't think you are using any...

If yo uwant to override a class try to play around with a selector value, here is a graphic that will help you understand

Reply:
![specificity-wars](https://github.com/CJ484/dogWalk/assets/7786627/23226111-2bae-491b-b334-a15f47668758)
Here is "Power/priority" of each selector and how it affects the styles

Reply:
I removed the !important from the file and overrided the bootstrap class